### PR TITLE
completed frontend structure criteria.

### PR DESCRIPTION
**Pull request by BxRay95**

### CODE REVIEW

File: src/components/Header.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  Collapse,
+  Navbar,
+  NavbarToggler,
+  NavbarBrand,
+  Nav,
+  NavItem,
+  NavLink,
+} from 'reactstrap';
+
+function Header(props) {
```
Comment (sjproctor) [inline src/components/Header.js]: This should be `const Header = () => {`

Are you using props?

File: src/App.css [hunk 2]
```diff
@@ -1,38 +1,20 @@
...
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
+.NotFound {
```
Comment (TsunamiCode5) [inline src/App.css]: This casing should be kebob not Pascal

**Review by TsunamiCode5 — CHANGES_REQUESTED** [review]
Take a look at the casing inside of your classNames and CSS file

**Review by TsunamiCode5 — APPROVED** [review]
NICE

**Pull request merged**